no-computer.md

```diff
@@ -58,12 +84,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Virtually ALL of the work is on the language side, as the idea is that the systems
+are already fixed.
```

Comment:
Is drawing the artwork language design? One could argue that "a smilee face" is language but the actual symbol is not, especially since many devices display them differently